.ci/validation.sh

```diff
@@ -1329,10 +1332,6 @@ website-only)
   ./mvnw -e --no-transfer-progress clean site -Pno-validations
   ;;
 
-pmd)
-  ./mvnw -e --no-transfer-progress clean test-compile pmd:check
```

Comment:
why not just use simple check like in `spotbugs` and `spotless`.

We already on happy path being complain. All we do is baby steps not on local dev env.